processing/data_collection/gazette/spiders/sp_marilia.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpMariliaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3529005"
+    name = "sp_marilia"
+    allowed_domains = ["marilia.sp.gov.br"]
+    start_urls = ["https://www.marilia.sp.gov.br/portal/diario-oficial"]
+
+    def parse(self, response):
+        gazettes = response.css(".d_e_modelo_diario")
+        for gazette in gazettes:
+            gazette_url = gazette.css("a::attr(href)").getall()[1]
```

Comment:
Relying on specific list indexes are not good. If they include a new `a` tag before that, it will break the spider, even if the entire structure remain the same. Try to be more specific on your selectors:

`gazette.xpath(".//a[contains(string(), 'Download')]/@href").get()` would be better to avoid this.